monitoring_service/constants.py

```diff
@@ -1,3 +1,4 @@
+EVENT_CHANNEL_OPEN = 'ChannelOpened'
```

Comment:
Can you change this to use the contract constants? https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/constants.py

Reply:
Done in 362b125212e6bebe85db364bc68f01a06571df64.